recipes-core/system-api-fake-fifo/system-api-fake-fifo.bb

```
@@ -13,7 +13,7 @@ inherit update-rc.d
 
 do_install() {
     install -d ${D}${sysconfdir}/init.d
-    install -m 0755 ${WORKDIR}/${INITSCRIPT_NAME} ${D}${sysconfdir}/init.d/${INITSCRIPT_NAME}
+    install -m 0755 ${WORKDIR}/init ${D}${sysconfdir}/init.d/${INITSCRIPT_NAME}
```

no, init does not exist anymore because you renamed it to system-api-fake-fifo. So this recipe will fail on a fresh build.
I would roll it back since you renamed the file. So the previous line would work now
```suggestion
    install -m 0755 ${WORKDIR}/${INITSCRIPT_NAME} ${D}${sysconfdir}/init.d/${INITSCRIPT_NAME}
```